test/checkPages_test.js

```diff
@@ -1462,5 +1462,27 @@ exports.checkPages = {
     testOutput(test,
       ['Page: file:test/validPage.html (00ms)'],
       []));
+  },
+
+  nonAscii: function(test) {
+    test.expect(7);
+    nock('http://example.com')
+      .head(encodeURI('/first/☺')).reply(200)
+      .get(encodeURI('/first/☺')).reply(200)
+      .head(encodeURI('/second/☺')).reply(200)
+      .get(encodeURI('/second/☺')).reply(200)
+      .head(encodeURI('/third/☺ ☺')).reply(200)
+      .get(encodeURI('/third/☺ ☺')).reply(200);
+    runTest({
+      pageUrls: ['test/nonAscii.html'],
+      checkLinks: true
+    },
+    testOutput(test,
+      ['Page: file:test/nonAscii.html (00ms)',
```

Comment:
Your comment in checkPages.js mentions a domain name with an e/accent character - it would be nice to include tests for international domains as well if that comes for free by using an npm package.